src/capture.js

```diff
@@ -439,11 +439,16 @@ Capture.ios8AndGreaterScrollFix = function(doc, callback) {
     // Using `getElementsByTagName` here because grabbing head using
     // `document.head` will throw exceptions in some older browsers (iOS 4.3).
     var head = doc.getElementsByTagName('head');
-    // Be extra safe and guard against `head` not existing.
-    if (!head.length) {
+    var viewport = doc.querySelector('meta[name="viewport"]');
+
+    // Be extra safe and guard against `head` not existing. We also don't need
+    // to do anything if the original source didn't have a viewport metatag.
```

Comment:
So, this is addressing the case where we're hitting an un-adapted page (we're already unmobifying). There's no viewport tag added through base.dust, since we didn't hit a route. Let's chat in person, might be easier to discuss.